www/FileTransfer.js

```diff
@@ -163,7 +162,7 @@ FileTransfer.prototype.upload = function(filePath, server, successCallback, erro
             }
         }
     };
-    exec(win, fail, 'FileTransfer', 'upload', [filePath, server, fileKey, fileName, mimeType, params, trustAllHosts, chunkedMode, headers, this._id, httpMethod]);
+    exec(win, fail, 'FileTransfer', 'upload', [filePath, server, fileKey, fileName, mimeType, params, false, chunkedMode, headers, this._id, httpMethod]);
```

Comment:
Do we need to keep the no-longer-used `trustAllHosts` parameter in the exec invocation? Also applies to the `download` method.